### DISCUSSION TITLE
Documentation: suitability as a regular search engine

### DISCUSSION THREAD

**Discussion by braco** [quickwit-oss/quickwit#3471] [General]
## My documentation idea

It seems like all of the messaging is about ingesting logs.

Elasticsearch is also very useful as a regular search engine against entity databases, like a movie database for example.

Is quickwit applicable to this kind of use? What are the pros and cons?

You can see vastly different, consumer-oriented messaging from Meili:

https://www.meilisearch.com/

**Comment by fmassot** [comment]
Hi @braco, that's a good question.

To put it simply, Quickwit is not built for user-facing search: typically e-commerce is not a good use case. Generally speaking, if you need a typo-tolerant search, a response time of a few milliseconds, or documents which are constantly being updated, then Quickwit won't be a good fit.

On the contrary, for search AND analytics on logs, traces, conversations, business events, and audit/security logs, Quickwit can handle very efficiently multi terabytes datasets. You can also have a look at this doc [page](https://quickwit.io/docs/overview/introduction#when-to-use-quickwit) to get more info.

If you are looking for criteria to choose your search engine, you need to look at all those different characteristics: performance, size, type of queries, mutability, etc.
